scripts/rebalance-queue-masters

```diff
@@ -6,6 +6,7 @@ set -o nounset
 # defaults
 declare -r default_vhost='/'
 declare -r default_queue_regex='.*'
+declare -ri default_disable_health_check=1
```

Comment:
Could you please use `true` and `false` strings instead? I would also like the default to be `false`

```bash
declare -r default_disable_health_check='false'
```